src/basilisp/lang/vector.py

```diff
@@ -119,13 +119,27 @@ def __len__(self):
         return len(self._inner)
 
     def __lt__(self, other):
+        """Return true if the SELF vector is shorter than the OTHER
+        vector, or the first unequal element in SELF when iterating
+        from left to right is less than the corresponding OTHER
+        element.
```

Comment:
```suggestion
        """Return true if the `self` vector is shorter than the `other`
        vector, or the first unequal element in `self` when iterating
        from left to right is less than the corresponding `other`
        element.
```

As a minor style point, I prefer to use single backticks to enclose names in the Python docstrings. For Clojure, we use double backticks because Sphinx uses reStructuredText to generate API docs.